test/commands/workflow/run.test.ts

```diff
@@ -60,7 +60,11 @@ describe("commands/workflow/run", () => {
                 "service-token": "valid-token",
                 "api-origin": undefined,
                 environment: "staging",
-                recipients: ["alice"],
+                recipients: [
```

Comment:
This comment is related to how we sent the data to the backend.
Right now we are sending the list as it is, and because JS lists has indexes we receive something like this on the backend:
``` "recipients" => %{
          "0" => %{"id" => "user-1"},
          "1" => %{"collection" => "projects", "id" => "object-1"}
        },
```

Maybe we should parse this and start sending a JSON instead, what do you think?